modules/user-profile-migration/src/main/java/org/apache/airavata/utils/InstallCert.java

```diff
@@ -0,0 +1,192 @@
+package org.apache.airavata.utils;
```

Comment:
@machristie,

Do we need this and if so what is it used for?
ans: I used this file to import iam.scigap.org's SSL certificate into JDK's keystore, it is not needed to run the tool.

Is the copyright notice on this file acceptable for inclusion in an Apache project?
Ans: Not Sure about this, It will be better if we remove this file because it is a Sun Systems Licensed file